module3/labs/week3/CreatingRelatedResources.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: Creating and Reading Related Resources Lab
+---
+
+## Setup
+
+For today's lab, we are going to continue building the Record Collection.
+
+**Objective**
+* Add Songs to our application
+* Use a form to create songs for each album
+
+* Open your Record Collection project.
+    * Run the project, and make sure you have the Record#Index, #Show, #New, and #Create actions working.
+* If you have been absent or can't find your project, follow these steps:
+    * Fork and Clone it from this [Starter Repository](https://github.com/turingschool-examples/LaunchFeatureLabStarter_RecordCollection).
+    * Build the project (let your instructor know if you receive any errors!).
+        * Open the database in pgAdmin, and create some records by running the following script (feel free to update the albums and artists to your favorites!).
+    ```sql
+    INSERT INTO albums (title, artist, release_date)
+    VALUES
+    ('Stop Making Sense', 'Talking Heads', '1984-06-24'),
+    ('Banana Winds', 'Jimmy Buffet', '1996-12-06')
+    ```
+    * Check out the branch _____________________ TODO need a branch for after REST & Create Single Resource lessons___________
+
+## Create the Song Model and Table
+
+1. With a partner,
+    * determine how a song relates to an album
+    * List the changes that need to be made to your application in order to add a song table to our database (using migrations!)
+1. Individually, implement the song model and table into your application
+    * Use your partner as a resource if you get stuck
+    * If you are both stuck, reach out to an instructor!
+1. When finishied, check in with an instructor **before** moving on
```

Comment:
I'm trying to think about how to make this flow well with two partners working independently at different paces. Because for the next section, they need to be there at the same point to create user stories together. Maybe you have something the quicker partner works on, like a codewards problem?